packages/ember/tests/routing/router_service_test/non_application_test_test.js

```diff
@@ -0,0 +1,110 @@
+import { inject as injectService } from '@ember/service';
+import { Router, NoneLocation } from '@ember/-internals/routing';
+import { get } from '@ember/-internals/metal';
+import { run } from '@ember/runloop';
+import { Component } from '@ember/-internals/glimmer';
+import { RenderingTestCase, moduleFor } from 'internal-test-helpers';
+
+moduleFor(
+  'Router Service - non application test',
+  class extends RenderingTestCase {
+    constructor() {
+      super(...arguments);
+
+      this.resolver.add('router:main', Router.extend(this.routerOptions));
+      this.router.map(function() {
+        this.route('parent', { path: '/' }, function() {
+          this.route('child');
+          this.route('sister');
+          this.route('brother');
+        });
+        this.route('dynamic', { path: '/dynamic/:dynamic_id' });
+        this.route('dynamicWithChild', { path: '/dynamic-with-child/:dynamic_id' }, function() {
+          this.route('child', { path: '/:child_id' });
+        });
+      });
+    }
+
+    get routerOptions() {
+      return {
+        location: 'none',
+      };
+    }
+
+    get router() {
+      return this.owner.resolveRegistration('router:main');
+    }
+
+    get routerService() {
+      return this.owner.lookup('service:router');
+    }
+
+    afterEach() {
+      super.afterEach();
+    }
+
```

Comment:
This doesn't `return super.afterEach()` so if the super classes `afterEach` returned a promise we wouldn't properly wait for the async, but ultimately since we don't have any customizations here, we can probably delete this (`super.afterEach()` would be called naturally).

```suggestion
```